Detectors/TRD/workflow/include/TRDWorkflow/TRDTrapSimulatorSpec.h

```diff
@@ -44,7 +44,7 @@ class TRDDPLTrapSimulatorTask : public o2::framework::Task
   int mShowTrackletStats = 1;        // show some statistics for each run
   bool mEnableOnlineGainCorrection{false};
   bool mEnableTrapConfigDump{false};
-  bool mShareDigitsManually{false}; // duplicate digits connected to shared pads if digitizer did not do so
+  bool mShareDigitsManually{true};  // duplicate digits connected to shared pads if digitizer did not do so
```

Comment:
`true` should be default in o2. Need to be set to false only for converted digits from Run 2